widgetssdk/src/main/java/com/glia/widgets/chat/controller/ChatController.java

```diff
@@ -119,6 +119,33 @@ public void error(GliaException exception) {
 
     private final GliaOperatorMediaRepository.OperatorMediaStateListener operatorMediaStateListener = this::onNewOperatorMediaState;
 
+    private final GliaSendMessageUseCase.Listener sendMessageCallback = new GliaSendMessageUseCase.Listener() {
+        @Override
+        public void messageSent(VisitorMessage message) {
+            onMessageSent(message);
+        }
+
+        @Override
+        public void onMessageValidated() {
+            emitViewState(chatState.chatInputChanged(""));
+        }
+
+        @Override
+        public void errorOperatorNotOnline(String message) {
+            onSendMessageOperatorOffline(message);
+        }
+
+        @Override
+        public void errorMessageInvalid() {
+
```

Comment:
can be removed if it is not used

Reply:
usecase calls it, we might need to handle it when error handling will be introduced

Reply:
or at least by the tests :)